db-client-java/src/main/java/com/eventstore/dbclient/AbstractPersistentSubscriptionSettingsBuilder.java

```diff
@@ -0,0 +1,141 @@
+package com.eventstore.dbclient;
+
+public class AbstractPersistentSubscriptionSettingsBuilder<T> {
+    protected int checkpointAfterMs;
+    protected boolean extraStatistics;
+    protected boolean resolveLinks;
+    protected boolean fromStart;
+    protected boolean fromEnd;
+    protected int historyBufferSize;
+    protected int liveBufferSize;
+    protected int maxCheckpointCount;
+    protected int maxRetryCount;
+    protected int maxSubscriberCount;
+    protected int messageTimeoutMs;
+    protected int minCheckpointCount;
+    protected int readBatchSize;
+    protected ConsumerStrategy strategy;
+
+    public AbstractPersistentSubscriptionSettingsBuilder() {
+        checkpointAfterMs = 2_000;
+        resolveLinks = false;
+        extraStatistics = false;
+        fromStart = false;
+        fromEnd = false;
```

Comment:
**FIX:** Based on the [RFC](https://github.com/EventStore/home/pull/550/files#diff-ad3a37ec8a5416ddcd905109248d13dba6a72a352ce7ee41b9d3d004aae1ff95R44) the default starting point should be the end.
```suggestion
        fromEnd = true;
```

Reply:
The default for streams is already defined to begin from the start, do we want to change this?
I changed it for the `all` stream subscription.

Reply:
I think that we should be aligned with the RFC also for the starting stream position.

Reply:
Done.